ui/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/process/ProcessId.scala

```diff
@@ -1,9 +1,10 @@
 package pl.touk.nussknacker.restmodel.process
 
+import io.circe.generic.JsonCodec
 import pl.touk.nussknacker.engine.api.process.ProcessName
 
-final case class ProcessId(value: Long) extends AnyVal
+@JsonCodec final case class ProcessId(value: Long) extends AnyVal
```

Comment:
I think it would be better to use deriveUnwrappedEncoder/deriveUnwrappedDecoder: 
https://github.com/circe/circe/pull/1427/files